packages/geoview-core/src/core/types/cgpv-types.ts

```diff
@@ -5,7 +5,7 @@ export * from '../components/app-bar/app-bar';
 export * from '../components/app-bar/app-bar-buttons';
 export * from '../components/app-bar/buttons/export';
 export * from '../components/app-bar/buttons/help';
-export * from '../components/app-bar/buttons/version';
+// export * from '../components/app-bar/buttons/version';
```

Comment:
I needed to comment out this line as it was the old version button, otherwise I used to get error in build that says `TypeAppVersion` has been exported already.